Getting-Started/Backoffice/Content-Templates/index.md

```diff
@@ -0,0 +1,66 @@
+# Content Templates
+Content Templates allow you to create a blueprint for new content nodes based on an existing node.
```

Comment:
Can we replace "you" with a role?  E.g. content editor?

Reply:
Hi @dampee I replaced this instance of you with the suggested 'content editor'. Did you want to replace all instances of 'you'?